tests/IpHelperTest.php

```diff
@@ -47,11 +47,8 @@ public function expandIpv6Provider(): array
 
     public function testIpv6ExpandingWithInvalidValue(): void
     {
-        try {
-            IpHelper::expandIPv6('fa01::1/64');
-        } catch (\Exception $exception) {
-            $this->assertStringEndsWith('Unrecognized address fa01::1/64', $exception->getMessage());
-        }
+        $this->expectException(\RuntimeException::class);
```

Comment:
Since there are two exceptions now, it makes sense to test for both cases.

Reply:
modified